src/aspire/source/xform.py

```diff
@@ -379,6 +428,9 @@ def __init__(self, xforms=None, memory=None):
         self.memory = memory
         self.active = True
 
+    def __str__(self):
+        return '\n'.join([f'{xform}' for xform in self.xforms])
```

Comment:
Specify that this is a `Pipeline` object. Also would be better to keep all the xforms on one line.

Reply:
Done in new commit a4a8fdc.